brave-clean-slate.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# This script removes Brave browser configuration, bookmarks, extensions and residual files.
+# Usage: ./brave-clean-slate.sh
+
+sudo rm -rf /Library/Application\ Support/BraveSoftware
+rm -rf ~/Library/Application\ Support/BraveSoftware
+
+# Residual files
+rm -rf /Library/Saved Application State/com.brave.Browser.savedState
+rm -rf /Library/Caches/BraveSoftware
```

Comment:
It would be nice to also `rm -rf ~/Library/Caches/com.brave.Browser`, `rm -rf ~/Library/Caches/com.brave.Browser.nightly`, `rm -rf ~/Library/Caches/com.brave.Browser.beta`.